src/pages/DatasetRegistration.js

```diff
@@ -151,9 +155,46 @@ class DatasetRegistration extends Component {
       prev.datasetData.datasetRepoUrl = datasetRepoUrl ? datasetRepoUrl.propertyValue : '';
       prev.datasetData.researcher = researcher ? researcher.propertyValue : '';
       prev.datasetData.pi = pi ? pi.propertyValue : '';
+      prev.datasetData.publicAccess = publicAccess;
+
+      return prev;
+    });
+
+    this.prefillDataUseFields(dataset.dataUse);
+  };
+
+  prefillDataUseFields(dataUse) {
+    let methods = dataUse.methodsResearch;
+    let genetics = dataUse.geneticStudiesOnly;
+    let publication = dataUse.publicationResults;
+    let collaboration = dataUse.collaboratorRequired;
+    let ethics = dataUse.ethicsApprovalRequired;
+    let geographic = dataUse.geographicalRestrictions;
+    let forProfit = !dataUse.commercialUse;
+    let hmb = dataUse.hmbResearch;
+    let poa = dataUse.populationOriginsAncestry;
+    let diseases = dataUse.diseaseRestrictions;
+    let other = !fp.isEmpty(dataUse.other);
+    let otherText = dataUse.other;
+    let generalUse = dataUse.generalUse;
+    // let pubRef = dataUse.pubRef;
```

Comment:
nit: do we need this?

Reply:
I've left this here because I don't actually see a "publication reference" string stored anywhere in the data use or dataset properties, although we wanted it displayed on the form.

Reply:
If we don't have a way to use `pubRef`, or store it in any way, then we should not be adding it to the code. Instead, leave it out and file a new ticket to "Add `pubRef` to register dataset page" so we don't lose track of it. We don't support `pubRef` currently, so it is not something we should be capturing in the UI.

Reply:
Done! https://broadinstitute.atlassian.net/browse/DUOS-844